src/main/java/core/basesyntax/ColoursDifferent.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum ColoursDifferent {
+    Pink,
```

Comment:
Enum values should be in uppercase, separated by underscores if needed. For example, PINK instead of Pink. 📚